### PR TITLE
Heystack 1366 homepage our work new section

### PR DESCRIPTION
**Pull request by brendankam1990**
## Tickets
[HEYSTACK-1366](https://app.clickup.com/t/9009201449/HEYSTACK-1366)

## Description
This update replaces the "Testimonial" section on the homepage with a "Featured Work" section, as shown in the screenshot below:

Screenshot 2024-11-29 at 11-45-18 Designing Inclusive   Accessible Solutions Hey Nova

## Changes Implemented
- Added a new **"Featured Work"** section to the homepage.
- Two featured works are displayed side by side, with randomized content that changes on refresh.
- A **Testimonial** section is displayed below the "Featured Work" section, featuring randomized testimonials on refresh.
- Added functionality for the **"View Client Page"** link to correctly redirect to the relevant partner's site.

## Testing Instructions
1. **Setup**:
   - Ensure you have a valid `.env` file configured to pull data from Airtable.
   - Switch to this branch and run `npm install`.

2. **Run the Application**:
   - Start the development server with `npm run develop`.
   - Navigate to `http://localhost:8000/` in your browser.

3. **Verify the Featured Work Section**:
   - Scroll down to the "Featured Work" section on the homepage.
   - Confirm that two featured works are displayed side by side.
   - Refresh the page and verify that two different works are displayed.
   - Ensure all images load correctly.

4. **Verify the Testimonial Section**:
   - Below the "Featured Work" section, confirm that a testimonial is displayed.
   - Refresh the page to verify that a new testimonial loads if available.
   - Refresh the page several times and check that the **"View Client Page"** link in the bottom-right corner correctly redirects to the partner's website.

## Notes
- Ensure both sections dynamically load content as expected.
- Verify that all images and links work seamlessly.

### CODE REVIEW

**Comment by kirstenjvd** [comment]
Task linked: [HEYSTACK-1366 Homepage "our work" new section](https://app.clickup.com/t/9009201449/HEYSTACK-1366)

**Comment by k-milbury** [comment]
I think it needs to collapse to smaller view a little bit earlier, around 851px it is super squished 
image

File: src/components/homepage/featuredTestimonial.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import React from "react"
+import styled from "@emotion/styled"
+import Img from "gatsby-image"
+import "./styles/styles.css"
+
+const TestimonialContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #00c7d4;
```
Comment (k-milbury) [inline src/components/homepage/featuredTestimonial.js:10]: This could probably be added as a variable

**Comment by k-milbury** [comment]
I would say as a general note too anywhere you're using a color value could probably be made into a variable incase the color theme changes. I also know a lot of the site in mainly in px but would be good to use rem/em (a lot is my fault from when I worked on it)

**Comment by brendankam1990** [comment]
* Added aspect ratio so it crops the image correctly.

Screenshot from 2024-12-02 11-39-47

* Also updated the testimonial section to collapse earlier.

**Review by k-milbury — APPROVED** [review]

**Pull request merged**